src/function/loadEmojione.js

```diff
@@ -12,16 +12,24 @@ function(emojione, uniRegexp, emojioneVersion, readyCallbacks, emojioneSupportMo
     return function(options) {
 
         function detectVersion(emojione) {
-            var version = emojione.cacheBustParam;
-            if (!isObject(emojione['jsEscapeMap'])) return '1.5.2';
-            if (version === "?v=1.2.4") return '2.0.0';
-            if (version === "?v=2.0.1") return '2.1.0'; // v2.0.1 || v2.1.0
-            if (version === "?v=2.1.1") return '2.1.1';
-            if (version === "?v=2.1.2") return '2.1.2';
-            if (version === "?v=2.1.3") return '2.1.3';
-            if (version === "?v=2.1.4") return '2.1.4';
-            if (version === "?v=2.2.7") return '2.2.7';
-            return '2.2.7';
+            var version;
+            if(emojione.cacheBustParam) {
+                version = emojione.cacheBustParam;
+                if (!isObject(emojione['jsEscapeMap'])) return '1.5.2';
+                if (version === "?v=1.2.4") return '2.0.0';
+                if (version === "?v=2.0.1") return '2.1.0'; // v2.0.1 || v2.1.0
+                if (version === "?v=2.1.1") return '2.1.1';
+                if (version === "?v=2.1.2") return '2.1.2';
+                if (version === "?v=2.1.3") return '2.1.3';
+                if (version === "?v=2.1.4") return '2.1.4';
+                if (version === "?v=2.2.7") return '2.2.7';
```

Comment:
How about 2.2.0, 2.2.1, 2.2.2, 2.2.3, 2.2.4, 2.2.5, 2.2.6 ?

Reply:
It was already like this in master. I just included version 3 in "else"